src/views/postUpdation/AllPostTypesUpdation.tsx

```diff
@@ -444,6 +479,31 @@ export const AllPostTypesUpdation: React.FC<IAllPostTypesUpdation> = ({
         disabled={isDisabled}
         post={post}
       />
+      <BorderBottom />
+      <h1>Changes List</h1>
+      <Box mt={3}>
+        <AuthorListDropdown
+          pageSizes={changesPerPage}
+          setChanges={handleNotesToShowChange}
+          selected={
+            size === 0 ? changesPerPage[changesPerPage.length - 1] : size
+          }
+        />
+        <List>
+          {changes.length > 0 ? (
+            changes.map((item) => {
```

Comment:
another approach is to render `changes.map` always:

```
<List>
  {changes.map(item => .....)}
  {changes.length === 0 && <>No Changes</>}
```

Definitely no difference in behaviour. But to me this variant is easier to follow, than ternary `? :`